### ISSUE TITLE
Patch on numeric fields from 0 to null

### ISSUE THREAD

**Issue by metas-lc** [metasfresh/metasfresh-webui-frontend#1883]
### Is this a bug or feature request?
bug
### What is the current behavior?
in one case, Patch on numeric fields is not sent
#### Which are the steps to reproduce?
1. go to product prices window (540325)
2. open any line from there, set a price
3. remove that price, press enter/tab/click somewhere else
=> notice that there are 2 patches: one for "" and one for "0" OK
4. now as the price is 0.00 , select it, remove it, and press enter/tab/click somewhere else
=> there is only the patch for "" NOK
### What is the expected or desired behavior?
price shall be set on 0

**Comment by siemiatj** [comment]
Two patches come from explicitly submitting the field, and bluring the field. But in my case the value is saved properly. Since we remove the value, and send no value (which is overwritten by response from the server with `0.00`) I'm not sure if we should be bothered by this if the final value is ok. Which in my case it is, but I might be doing something wrong here.

@teosarca wdyt ?

![c6id7tn6a4](https://user-images.githubusercontent.com/513460/42886669-70234c98-8aa4-11e8-826e-92a750369851.gif)

**Comment by teosarca** [comment]
@siemiatj  @metas-lc
did a quick check and IMHO that's OK, i.e.
* frontend is sending `""` 
* when the record is queried back, the "0.00" is returned for that field

BUT, what i see not OK in your screenshot/GIF is that `/edit` is called twice (tried on w101 and both of those calls are PATCH and not one OPTIONS and one PATCH), so as a result `/byIds` is called twice.

![image](https://user-images.githubusercontent.com/1244701/43123944-5ff5a72e-8f2e-11e8-8cfb-f2ad2348e377.png)

**Comment by siemiatj** [comment]
Yeah I'm aware of that, but we'll need a better test coverage before I can change this. As I've said before - patches come from editing value and blurring the field. And fixing this to only patch once will be really tricky.

**Comment by siemiatj** [comment]
Retest after #2026 is merged. Maybe my quick fix there will work across the board.
